From d5eaa8be998da2430ffd43e0eefa617cf1550ca3 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Thu, 31 Jan 2019 18:19:30 +0100
Subject: [PATCH] Install the FindQuaZip.cmake in the right prefix

It should go wherever CMAKE_INSTALL_PREFIX is installing.
This fixes the build on flatpak and should also fix it for other cross
compilation setups.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git CMakeLists.txt CMakeLists.txt
index 539caa4..a43fefe 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -60,4 +60,4 @@ set(QUAZIP_LIB_TARGET_NAME quazip${QUAZIP_LIB_VERSION_SUFFIX} CACHE
 
 add_subdirectory(quazip)
 
-install(FILES FindQuaZip.cmake RENAME FindQuaZip${QUAZIP_LIB_VERSION_SUFFIX}.cmake DESTINATION ${CMAKE_ROOT}/Modules)
+install(FILES FindQuaZip.cmake RENAME FindQuaZip${QUAZIP_LIB_VERSION_SUFFIX}.cmake DESTINATION share/cmake)
